src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -679,6 +679,7 @@ private static MemberInfo GetMemberInfo<T>(Expression<T> expr)
 
         private static Task ExecuteTask<T>(Task<T> task, HttpContext httpContext)
         {
+            EnsureRequestTaskOfNotNull(task);
```

Comment:
Nit: Let's add newlines before all the local functions. Or just put this after the local function definition (again with newlines surrounding it. Same goes for all the other `Ensure` method calls.

```suggestion
            EnsureRequestTaskOfNotNull(task);

```